examples/plot_geodesics_h2.py

```diff
@@ -12,7 +12,7 @@
 import geomstats.visualization as visualization
 from geomstats.geometry.hyperbolic import Hyperbolic
 
-H2 = Hyperbolic(dimension=2)
+H2 = Hyperbolic(dimension=2, point_type='extrinsic', scale=1)
```

Comment:
Is this necessary? The default value of scale is 1, right?

Reply:
No, it was not necessary.